Django models `__str__` codemod (#302)
pixee/codemodder-python: src/core_codemods/docs/pixee_python_django-model-without-dunder-str.md

> If you've ever actively developed or debugged a Django application, you may have noticed Django models and their instances can sometimes be hard to read or distinguish one instance from another. Loading models in the interactive Django console or viewing them in the admin interface can be puzzling. This is because Django is trying to display your model objects as a plain strings.

Reviewer: ```suggestion
If you've ever actively developed or debugged a Django application, you may have noticed that the string representations of Django models and their instances can sometimes be hard to read or to distinguish from one another. Loading models in the interactive Django console or viewing them in the admin interface can be puzzling. This is because the default string representation of Django models is fairly generic.
```

> We've written this codemod to make your model objects human-readable. It will automatically detect all of your model's fields and display them as a nice string.

Reviewer: ```suggestion
This codemod is intended to make the string representation of your model objects more human-readable. It will automatically detect all of your model's fields and display them as a descriptive string.
```

> For example, the `Question` model from Django's popular Poll App tutorial will look like this:

Reviewer: ```suggestion
For example, the default string representation of the `Question` model from Django's popular Poll App tutorial looks like this:
```

```diff
from django.db import models

class Question(models.Model):
    question_text = models.CharField(max_length=200)
    pub_date = models.DateTimeField("date published")
+ 
+    def __str__(self):
+        model_name = self.__class__.__name__

> +        fields_str = ", ".join([f"{field.name}={getattr(self, field.name)}" for field in self._meta.fields])

Reviewer 2: Change this example to reflect the change in the other comment.

+        return f"{model_name}({fields_str})"
```

> Without this change, the `Question` objects look like this in the interactive Django shell:

Reviewer: ```suggestion
Without this change, the  string representation of `Question` objects look like this in the interactive Django shell:
```
